app/src/main/res/values/themes.xml

```diff
@@ -224,4 +225,11 @@
         <item name="android:textColorSecondary">?bottomBarIconColor</item>
     </style>
 
+    <style name="DaxAlertDialog" parent="ThemeOverlay.MaterialComponents.MaterialAlertDialog">
```

Comment:
more info on these fields and Dialog styling can be found [here](https://github.com/material-components/material-components-android/blob/master/docs/components/Dialog.md)